app/src/main/java/fr/free/nrw/commons/bookmarks/locations/BookmarkLocationsDao.java

```diff
@@ -287,6 +292,13 @@ public static void onUpdate(SQLiteDatabase db, int from, int to) {
                     Timber.e(exception);
                 }
             }
+            if (from == 14){
```

Comment:
This should have been 13, since we are coming from 13 not 14. And we will go to 14 not 15.

Reply:
Understood 
Thanks @neslihanturan .